src/sentry/api/endpoints/relay_register.py

```diff
@@ -184,8 +184,15 @@ def post(self, request):
                 relay_id=relay_id, public_key=public_key, is_internal=is_internal
             )
         else:
-            relay.last_seen = timezone.now()
             relay.is_internal = is_internal
             relay.save()
 
+        try:
+            relay_usage = RelayUsage.objects.get(relay_id=relay_id, version=version)
+        except RelayUsage.DoesNotExist:
+            RelayUsage.objects.create(relay_id=relay_id, version=version)
+        else:
+            relay_usage.last_seen = timezone.now()
+            relay_usage.save()
```

Comment:
You could potentially use `create_or_update` here, since you don't need a reference to the object afterwards.